webpack.playground.config.js

```diff
@@ -11,14 +11,16 @@ const WIDGET_RC = '.widgetrc';
 
 let widgetRc = {
   widgetOptions: {
-    baseUrl: 'https://{yourOktaDomain}',
+    baseUrl: 'http://localhost:3000',
```

Comment:
Right now we are having tests for idx pipeline powered by mock server right ? And if no widgetrc is provided by default playground opens up with idx flow pointing to mock server right.

Reply:
correct.